### ISSUE TITLE
export-policy does not retain "Path" value

### ISSUE THREAD

**Issue by cmckendry** [tilfin/aws-iam-policy-tool#5]
When exporting policies via awsiamtool, the resulting json files do not contain a value for the "Path" property of the IAM policy. For example:

```
# cat ./dibs-iam/AWSLambdaBasicExecutionRole-6637c766-dc66-4cb3-b99f-ea9896ddcedb.json
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Effect": "Allow",
            "Action": "logs:CreateLogGroup",
            "Resource": "arn:aws:logs:us-east-1:268215509542:*"
        },
        {
            "Effect": "Allow",
            "Action": [
                "logs:CreateLogStream",
                "logs:PutLogEvents"
            ],
            "Resource": [
                "arn:aws:logs:us-east-1:268215509542:log-group:/aws/lambda/1stdibs-ops-prod-url-shortener:*"
            ]
        }
    ]
}
```

Contrast this with the output of the same policy using the `awscli` iam tools:

```
# aws iam get-policy --policy-arn arn:aws:iam::268215509542:policy/service-role/AWSLambdaBasicExecutionRole-6637c766-dc66-4cb3-b99f-ea9896ddcedb
{
    "Policy": {
        "PolicyName": "AWSLambdaBasicExecutionRole-6637c766-dc66-4cb3-b99f-ea9896ddcedb",
        "PermissionsBoundaryUsageCount": 0,
        "CreateDate": "2019-04-24T19:14:08Z",
        "AttachmentCount": 1,
        "IsAttachable": true,
        "PolicyId": "ANPAT44XD4ITDYVYFFQMT",
        "DefaultVersionId": "v1",
        "Path": "/service-role/",
        "Arn": "arn:aws:iam::268215509542:policy/service-role/AWSLambdaBasicExecutionRole-6637c766-dc66-4cb3-b99f-ea9896ddcedb",
        "UpdateDate": "2019-04-24T19:14:08Z"
    }
}
```

This is being observed with an autogenerated execution role created for a lambda function, so I'm assuming it's not a wildly uncommon or pathological use case. 

This then causes issues down the line, as `validate-policy` is not able to correctly associate the json file with the corresponding IAM policy:

```
# awsiamtool validate-policy dibs-iam
 NG  AWSLambdaBasicExecutionRole-6637c766-dc66-4cb3-b99f-ea9896ddcedb does not exist.
```

At first principles it's not clear to me what the desired behavior for this case is. The obvious solution would be to create literal directories in the file system so that the json file path matches the IAM policy "Path" but I can also see how having a nested file structure could easily cause undesired side effects. Hoping you can provide some additional perspective on this.

Thanks!

**Comment by tilfin** [comment]
The idea behind this tool is to help you manage the IAM roles and policies strictly. Therefore we assume that we will define the policy at first. They should be out of the management target because service-policy is automatically generated.

However, in practice, when we introduce this tool in the middle of a project, we feel it difficult, so I would like to modify it.

**Comment by tilfin** [comment]
released 0.7.0 for new policy definition.
